pkg/registry/rbac/rolebinding/policybased/storage.go

```diff
@@ -47,12 +47,17 @@ func (s *Storage) Create(ctx genericapirequest.Context, obj runtime.Object) (run
 		return s.StandardStorage.Create(ctx, obj)
 	}
 
+	namespace, ok := genericapirequest.NamespaceFrom(ctx)
```

Comment:
This looks weird when you're reading it.  Explain why we pull the namespace out here.